test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests/Utility/AuditUtilityTests.cs

```diff
@@ -431,6 +453,7 @@ await createGraphTasks[1]
             "/path/proj.csproj",
             graphs,
             vulnerabilityProviders,
+            targetFrameworks: Enumerable.Empty<TargetFrameworkInformation>().ToList(),
```

Comment:
```suggestion
            targetFrameworks: new List<TargetFrameworkInformation>(0),
```

The purpose of `Enumerable.Empty<T>()` is to get an empty enumerable that doesn't cause any heap allocations (after the first time it's called). However, using `ToList()` defeats that purpose, as it has to allocate a `List<T>`. Using LINQ's `ToList` is just a less efficient way of creating the list directly, so might as well make the line shorter as well.

However, as mentioned in a comment in the production code, this is invalid from a business point of view. This list cannot ever be empty when the restore `graphs` are non empty. They must have the same TFMs, after all the restore graphs are calculated from the input target frameworks.